maintainers/maintainer-list.nix

```diff
@@ -7834,6 +7834,12 @@
     githubId = 42545625;
     name = "Maas Lalani";
   };
+  maddiethecafebabe = {
```

Comment:
Please extract this change to a separate commit, see https://github.com/NixOS/nixpkgs/pull/183559 for an example.